__test__/extensions/service-vendors/twilio.test.js

```diff
@@ -841,7 +841,6 @@ describe("config functions", () => {
               ["TWILIO_ACCOUNT_SID", organization, expectedConfigOpts]
             ]);
             expect(configFunctions.getConfig.mock.calls).toEqual([
-              ["TWILIO_AUTH_TOKEN_ENCRYPTED", organization, expectedConfigOpts],
               ["TWILIO_AUTH_TOKEN_ENCRYPTED", organization, expectedConfigOpts],
               ["TWILIO_ACCOUNT_SID", organization, expectedConfigOpts],
               ["TWILIO_ACCOUNT_SID", organization, expectedConfigOpts],
```

Comment:
Following the logic of these changes, and wondering if the duplicate TWILIO_ACCOUNT_SID should also be deleted?

Reply:
No, the change to this test was made because of the change in `src/extensions/service-vendors/twilio/index.js`. The change in `src/extensions/service-vendors/twilio/index.js` only deals with `TWILIO_AUTH_TOKEN_ENCRYPTED`, not `TWILIO_ACCOUNT_SID`, so you will notice if you remove the duplicate `TWILIO_ACCOUNT_SID`, the test will fail.